_includes/volunteer/success.html

```diff
@@ -1,41 +1,37 @@
 <div class="jumbotron background-white">
-    <div class="row" style="padding-bottom:20px;">
+    <div class="row padding-bottom-md">
         <div class="col-md-9">
             <h2>Next Steps</h2>
         </div>
     </div>
     <ol>
         <li class="successHeaders">
-            <div class="row" style="display: flex;
-            flex-wrap: wrap;">
-               <!-- <button class="navbar-toggler collapsed hamburger-background light-purple-background align-left buttonSlack" type="button" onclick="redirect()" >
-                    <i  class="fab fa-slack volunteerSlack" style="color:#9933CC; "></i>
-                </button> -->
+            <div class="row d-flex flex-wrap">
                 <div class="col-s-4 col-xs-4" >
-                    <button class="navbar-toggler collapsed light-purple-background align-left buttonSlack" style="margin-left:20px;" type="button" onclick="redirect()" >
-                        <i  class="fab fa-slack volunteerSlack" style="color:#9933CC; "></i>
+                    <button class="navbar-toggler collapsed light-purple-background align-left buttonSlack margin-left-lg" type="button" onclick="redirect()" >
+                        <i  class="fab fa-slack volunteerSlack success-pink"></i>
                     </button>
                 </div>
                 <div class="col-s-8 col-xs-8">
-                    <a class="successHeaders" style="display:inline; color:#9933CC;margin-left:10px; " href="http://slack.cantstopcolumbus.com/" target="_blank">Request to join our slack</a>
+                    <a class="successHeaders success-link margin-left-sm "  target="_blank">Accept our incoming invitation to our Slack Channel!</a>
```

Comment:
is this a link to nowhere? Could be confusing to the users if so. Should we change this so it doesn't look like a link?

Reply:
That's a good idea...let me see what I can do